### DISCUSSION TITLE
イベント会場 Symbolの採用アートを作ろう

### DISCUSSION THREAD

**Discussion by ymuichiro** [ymuichiro/symbol_japan_forum#32] [General]
以下の募集がありました
作品の掲載や問いかけ、応援などこちらでお願いします

https://github.com/ymuichiro/symbol_japan_forum/issues/25#issuecomment-1114280326

**Comment by ymuichiro** [comment]
応募者：ふぁー

![Quest](https://user-images.githubusercontent.com/47295014/166414524-011c8c40-378c-44ed-bb20-72bae3c3ef5b.png)

**Comment by yasudagabou** [comment]


**Reply by ymuichiro** [reply]
おお、きれい＼(^o^)／

**Reply by yasudagabou** [reply]
シンボルカラーのライティングにホタルを加えて幻想的な雰囲気を出してみました

**Reply by yasudagabou** [reply]
![掲示板夜](https://user-images.githubusercontent.com/105069276/167233336-8d9e8e47-02ff-49df-822a-8620d9e7041e.jpg)

**Reply by yasudagabou** [reply]
ロゴ装飾が見えにくく感じたので森の夜版はこちらでエントリーさせてください

**Reply by ymuichiro** [reply]
@yasudagabou こちら入賞しました！お手数ですがTwitterのDMで私宛に賞金送付用のSymbol Address貰っても良いですか。何位入賞かは別途発表致します。

**Reply by ymuichiro** [reply]
すみません。matsunokiさんと連絡被りました。matsunokiさんに送っていたら私の方にはもらわなくとも大丈夫です

**Reply by yasudagabou** [reply]
matunokiさんにおくりました。　ちなみに発表はいつ頃になるのでしょうか？

**Reply by ymuichiro** [reply]
@yasudagabou ちょっと対応中のハチェットさんに進捗聞いてみますね

**Reply by ymuichiro** [reply]
ちょっとハチェットさん最近別件で捕まりすぎて、これの件着手が遅れているようです
すみませんが、ちょっと続報お待ちください。時間を空けて改めて確認してみます。

**Comment by yasudagabou** [comment]
![掲示板](https://user-images.githubusercontent.com/105069276/167179990-8a2fc039-ada1-41f1-8a91-9f33a4b156ae.jpg)

**Comment by yasudagabou** [comment]
![掲示板３](https://user-images.githubusercontent.com/105069276/167232783-7925c7d3-1191-46be-951b-29e47bb50244.jpg)

**Comment by MTNsyndicate** [comment]
応募者/@MTN_syndicate

草刈り中に考えましたw

![image](https://user-images.githubusercontent.com/103474958/167262939-64e8941d-2cad-45c3-8cb9-d8355404a8fe.jpeg)

**Reply by ymuichiro** [reply]
おお、近未来

**Reply by ymuichiro** [reply]
@MTNsyndicate こちら入賞しました！お手数ですがTwitterのDMで私宛に賞金送付用のSymbol Address貰っても良いですか。何位入賞かは別途発表致します。

**Reply by ymuichiro** [reply]
すみません。matsunokiさんと連絡被りました。matsunokiさんに送っていたら私の方にはもらわなくとも大丈夫です

**Reply by MTNsyndicate** [reply]
ご連絡ありがとうございます。

今朝マツノキさんよりDMを頂いた際に自分のウォレットアドレスをお伝えしました。

**Reply by ymuichiro** [reply]
ちょっとハチェットさん最近別件で捕まりすぎて、これの件着手が遅れているようです
すみませんが、ちょっと続報お待ちください。時間を空けて改めて確認してみます。

**Comment by ymuichiro** [comment]
応募者： [みー](https://twitter.com/MnkDrD_M)
![みーさん_MnkDrD_M](https://user-images.githubusercontent.com/47295014/167298932-248b30b2-a06e-4016-b3d3-cd4b755f8e00.jpeg)

**Reply by ymuichiro** [reply]
Twitterアカウント名：みー🐯
TwitterID：@MnkDrD_M

**Comment by ymuichiro** [comment]
ちょっと本件、お金の管理回りで調整が入っているらしく、時間かかっているようです
投稿頂いた方、申し訳ありませんが、ハチェットさんから続報あり次第、共有致します

**Reply by yasudagabou** [reply]
恐れ入りますがコメント失礼します。上のアナウンスより１ヶ月経ってしまいましたが、続報がないようであれば問い合わせていただいてよろしいでしょうか？　神経質に感じられてしまったらすみませんが、皆さん手間を掛けて作成したのに「軽視されているのかな」と思われてしまったら運営としても残念なことじゃないかなと思いますので

**Reply by koooo0** [reply]
連絡きたので見てみた、クリエイターさんに先払いして運営内で後始末を行えばOKかと！ｗ
割と企画にありがちな問題

**Reply by ymuichiro** [reply]
取り急ぎ、あの後から私からも週一くらいで状況どうですかは聞いてみています
ちょっとずつ送金手続きをしているニュアンスの返信は来ていますが、まだ手続き完了連絡はこないですね

ちなみに今日ちょうど mikun さんも問いかけてくれました。
今あちらは早朝だと思うので明日か明後日返信が来るかのぞいてみようと思います

https://discord.com/channels/856325968096133191/967721981199921172/989044599433596970

**Reply by ymuichiro** [reply]
ちなみに返事きた。ハチェットさん、精神的にお疲れ？

https://discord.com/channels/856325968096133191/967721981199921172/989622796382515210

**Reply by yasudagabou** [reply]
週一で聞かれてるのですね、　かしこまりました。

**Reply by yasudagabou** [reply]
koooooo<
順位など細かいことどう決まってるか（ひょっとしたら入賞連絡だけで決まってないのかもですが）分からないので動けない・動きづらいのかもですが、　私はもちろん他の参加者も気になってる可能性もあるのでコメントした次第です。

**Reply by matsuno3** [reply]
こんにちは。
いつもご活動拝見しております。
今回、応募していただいた皆様には大変ご心配おかけしております。
端的に言うと、仁義を通していないコアデブに問題がありますので、結論がでるまで対話を続けていきます。
私は公示から、現在に至るまで把握しております。
また、ふぁーさん含め見えないところで動いていましたが、公表できるものは今後公表していきます。

今回、依頼者の意図、目的、現状を対話により明確にしていきますが、ハチェット氏が現状体調不良の為対話ができない状態になっております。コアデベロッパーはSymbolの顔であり今回の事象は決して印象の良いものではありません。
まずは、今回の状況の意図を聞き出しております。長期間になるかもしれませんが、理由が明確にならないと後味が悪いのでどうぞご協力、ご理解のほどよろしくお願いいたします。

**Reply by matsuno3** [reply]
こんばんは。まずは、そもそも発表されるはずだった結果に関して問い合わせを行いました。
若干センシティブなやりとりなので、割愛させていただきます。どうしても知りたい応募者さんは私にTwitterDMください
。知る権利はありますので。しかし知らないほうが良いこともあるのも事実です。第三者の方はご遠慮ください。

順位に関しては
１．Yasudagabou　さん
２．MTN　さん
３．Faunsu　さん
選考外　みーさん
ということです。

賞金に関してですが、
９日前に手続きを行い現状延滞しており、ハチェット氏の中では延滞の許容範囲であるということでした。
その前に手続きができなかったのは、取引口座のあるサービスシステムがトラブルを起こしていたという点、ハチェット氏が
体調悪化の為動けなかった点が原因とみられます。
しばらく待って様子を見てみようと思います。
以上　進捗のご報告でした。

選考外　みーさんの画像に関してはもったいないので広報窓口のヘッダーとしてユーザーにアンケートを取り、採用されれば使用させていただければと思います。

**Reply by ymuichiro** [reply]
ちなみに、私応募するつもりはなかったので（見本のつもり）みーさん良かったら賞金来たら転送してもよい…？

**Reply by matsuno3** [reply]
特に私としては問題ないという認識ですが、当事者同士で話し合って決めていただければ幸いです。

**Reply by yasudagabou** [reply]
何度も問い合わせありがとうございました。　私自身それほどメンタルに自信ないので詳細知らないほうがいいことであればそのままでいようと思います。今後仕事にしてもコンテストにしても、より相手の求めるものを考えて描いていこうと思います。

**Reply by matsuno3** [reply]
こんばんは。
前日ハチェット氏から別途コメントあり、私が対応しますというコメントが最新になっています。
以上　よろしくお願いいたします。

**Reply by matsuno3** [reply]
もし１４日以上たってもアクションがない場合、再度お話に行きますのでご連絡ください。もしくは確認させていただきます。

**Reply by ymuichiro** [reply]
これの件、多分まだ入金はないと思っているんだけども、ここで一旦投稿してくださった方の感覚を聞いておきたいなと思っています。
（ちなみにもしかしてもう入金はされている？

暫く様子を見ますか？（もう1ヶ月と言わず、3ヶ月くらい）
一度間を空けるという選択肢もあるのかなと思いました

**Reply by matsuno3** [reply]
本日各応募者の送金、着金が確認できました。
皆様ご理解ご対応ありがとうございました。
この件はこれで終了とさせていただきます。

**Reply by MTNsyndicate** [reply]
いろいろとご対応頂き、ありがとうございました🙇‍♂️
着金確認できております。
